osp/views/answer.py

```diff
@@ -0,0 +1,79 @@
+from rest_framework import viewsets, status
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from osp.models import (
+    Question, Answer, ChoiceValue, CheckboxValue, DropdownValue, DateValue,
+    TimeValue, FileUploadValue, ShortAnswerValue, ParagraphValue, FormFeedback
+)
+from osp.serializers.answer import AnswerSerializer
+from osp.serializers.form_feedback import FormFeedbackSerializer
+from osp.utils.answer_function import get_model_and_serializer
+
+class AnswerView(viewsets.ModelViewSet):
```

Comment:
The list endpoint in this viewset is okay, but the create method needs to be moved to the formfeedback viewset because we need a single point where the form submissions are created